src/psyclone/psyir/nodes/omp_directives.py

```diff
@@ -587,9 +633,13 @@ class OMPTaskloopDirective(OMPRegionDirective):
                              a grainsize and num_tasks value \
                              specified.
     '''
+    _children_valid_format = ("Schedule, [OMPGrainsizeClause | "
```

Comment:
> I think I'd rather keep the ordering (and mutual exclusion)

That's fine by me but please at a comment like: "This specification respects the mutual exclusion of OMPGrainsizeClause and OMPNumTasksClause but adds an ordering requirement. Other specifications to soften the ordering requirement are possible but need additional checks in the global constraints"

Reply:
Done